docs/content/integrations/dbt-cloud.mdx

```diff
@@ -130,6 +130,21 @@ defs = Definitions(
 )
 ```
 
+## Step 4: Set up a GitHub action to cache your dbt Cloud job's compilation
+
+<Note>This step is optional.</Note>
+
+In Step 2, your dbt project is compiled when your Dagster definitions are first loaded. Because this compilation step occurs at runtime, you may encounter high latency when you load your code locations as you wait for the compilation to complete. Instead of compiling at runtime, a cached compilation can be used. A reference to this cached compilation can be created using a GitHub action on your dbt Cloud project.
```

Comment:
"at runtime" isn't the most precise description here -- maybe more like "when your repository is loaded"